monkey/agent_plugins/exploiters/ssh/src/ssh_command_builder.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import PurePath
+from typing import Sequence
+
+from common import OperatingSystem
+from common.common_consts import AGENT_OTP_ENVIRONMENT_VARIABLE
+from common.types import AgentID
+from infection_monkey.exploit import IAgentOTPProvider
+from infection_monkey.model import MONKEY_ARG
+from infection_monkey.utils.commands import build_monkey_commandline
+
+
+def build_ssh_command(
+    agent_id: AgentID,
+    servers: Sequence[str],
+    current_depth: int,
+    operating_system: OperatingSystem,
+    remote_agent_binary_destination_path: PurePath,
+    otp_provider: IAgentOTPProvider,
+) -> str:
+    if operating_system != OperatingSystem.LINUX:
+        raise Exception(f"Unsupported operating system: {operating_system}")
+
+    set_otp_command = f"{AGENT_OTP_ENVIRONMENT_VARIABLE}={otp_provider.get_otp()} "
+    cmdline_arguments = build_monkey_commandline(agent_id, servers, current_depth + 1)
+
+    cmdline = (
+        f"{set_otp_command} "
+        f"{remote_agent_binary_destination_path} {MONKEY_ARG} {cmdline_arguments} "
+        "> /dev/null 2>&1 &"
+    )
```

Comment:
I think you can use `infeciton_monkey.utils.commands.build_run_command` for this. It is worth checking it out